public/scripts/worktrollyUI.js

```diff
@@ -164,13 +164,14 @@ $("#submitCreateNewTask").click(function() {
     var createNewTaskFunction = firebase.functions().httpsCallable('createNewTask');
     createNewTaskFunction({ Title: title, Description: des, Priority: priority, Difficulty: difficulty, Creator: creator, Assignee: assignee, EstimatedTime: estimatedTime, Status: status, Category: category, SprintNumber: sprintNumber, StoryPointNumber: storyPointNumber }).then(result => {
         console.log(result.data);
-        newPage = "dashboard";
-        uiLoader();
+        // newPage = "dashboard";
+        // uiLoader();
+        location.reload();
     });
-    $('input').val('');
-    $('select').val('');
-    $('textarea').val('');
-    setInterval(setDataIntoDashboard, 1000);
+    // $('input').val('');
+    // $('select').val('');
+    // $('textarea').val('');
+    // setInterval(setDataIntoDashboard, 1000);
```

Comment:
remove all the commented lines.
And do the same thing in every form like logWork, editPage,Taskdescription.....